www/src/views/shared/collapsible.js

```diff
@@ -1,83 +1,73 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import { Component } from "react"
+import { useState } from "react"
 import { FaAngleDown, FaAngleUp } from "react-icons/fa"
 
-class Collapsible extends Component {
-  state = {
-    collapsed: false,
-  }
+import { Flex } from "theme-ui"
 
-  handleClick = () => {
-    this.setState({ collapsed: !this.state.collapsed })
-  }
+const Collapsible = props => {
```

Comment:
Turn this into a named function instead of a closure:

https://github.com/gatsbyjs/gatsby/issues/23575

Reply:
(This goes for the other PRs as well)

Reply:
This is not an anonymous function, it is already a named function, just the difference is it is arrow function instead of normal function, but considering the point mentioned in here https://github.com/gatsbyjs/gatsby/issues/23575#issuecomment-621162331, I will convert it into normal javascript function, also I will do the same for the other PR's.

Reply:
Huh! I did not know that.

At the very least, `function` notation means we can do `export default function MyComponent` instead of

```
const MyComponent = () => {}
export default MyComponent
```